### DISCUSSION TITLE
3D Export?

### DISCUSSION THREAD

**Discussion by benthroop** [graphia-app/graphia#43] [Ideas]
Is there any facility to export a graph in a standard 3D format like FBX or OBJ? If not, has there been any discussion about this? Thanks!

**Comment by timangus** [comment]
Sorry, no that's not possible at the moment. It's something we're asked reasonably regularly though, so perhaps I'll look at that at some point in the future. For the time being you can export the graph (to a variety of formats) and you can also export the node positions (from the *Layout* menu), so you could potentially write a script in your language of choice to bring these elements together that outputs a 3D model.
